starters/features/builder.io/src/components/gauge/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { component$ } from "@builder.io/qwik";
+import styles from "./gauge.module.css";
+
+export default component$(({ value = 50 }: { value?: number }) => {
+  const safeValue = value < 0 || value > 100 ? 50 : value;
```

Comment:
if value is NaN it will stay NaN :) You should do `value >=0 && value <=100?value:50`

Reply:
Sure @wmertens! Thank you!

Reply:
Done!